prime-router/metadata/hl7_mapping/receivers/STLTs/CA/ca-specimen.yml

```diff
@@ -64,6 +64,6 @@ elements:
     hl7Spec: [ '%{hl7SpecimenFieldPath}-18' ]
 
   - name: specimen-received-time-diagnostic
-    condition: '%resource.receivedTime.exists().not() and %diagnostic.issued.exists() and %diagnostic.issued is instant'
-    value: [ '%diagnostic.issued.changeTimezone(%timezone)' ]
+    condition: '%resource.receivedTime.exists().not() and %resource.collection.collected is dateTime'
+    value: [ '%resource.collection.collected(%timezone)' ]
```

Comment:
Good catch, I guess I didn't catch this because receivedTime will always get set by the default sender transform, but it might be safe to add it here too.